openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -3,8 +3,35 @@
 
 Includes namespacing, caching, and course overrides for waffle flags.
 
-For testing WaffleFlags, see testutils.py.
-For testing WaffleSwitchNamespace, use its context manager.
+Usage:
+
+For Waffle Flags, first set up the namespace, and then create flags using the
+namespace.  For example:
+
+    WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(namespace='course_experience', log_prefix=u'Course Experience: ')
```

Comment:
Nit: reading this, I wonder if the argument should be ``name`` not ``namespace``. It seems odd to pass a namespace to a namespace! I'm also wondering if we should drop the log_prefix, and just use the namespace. It seems unduly complicated to think about logging. At a minimum I'd default it from the provided name.